WordPress/Classes/Services/RawBlockEditorSettingsService.swift

```diff
@@ -31,7 +31,7 @@ final class RawBlockEditorSettingsService {
     /// the network.
     func getSettings(allowingCachedResponse: Bool = true) async throws -> Data {
         // Return cached settings if available
-        if allowingCachedResponse, let cachedSettings = try await BlockEditorCache.shared.getBlockSettings(for: blogID) {
```

Comment:
If local cache read using `try Data(contentsOf: fileURL)` were ever to fail, it would prevent `fetchSettingsFromAPI` from running. The cache lookup should not stop the request.